.github/workflows/release_portable_linux_packages.yml

```diff
@@ -181,6 +191,16 @@ jobs:
           --include "*.whl" \
           --include "*.tar.gz"
 
+      - name: Upload Releases to Staging S3
```

Comment:
```suggestion
      - name: Upload releases to staging S3
```

Reply:
Having this step _after_ `Upload Releases to S3` is confusing. I understand that copies to the release bucket right now happen without further testing and that there is more to do but the order can already be revised here.